### ISSUE TITLE
[Jakarta EE 9.1 landing page] Add the WG blog announcement to the page

### ISSUE THREAD

**Issue by aeiras** [jakartaee/jakarta.ee#1186]
from https://github.com/jakartaee/jakarta.ee/issues/1177#issuecomment-843420505
can we add the feature that houses the WG written announcement of the release to the page?

**Comment by autumnfound** [comment]
@aeiras It looks like a few things have been merged around this issue. Has this been solved?

**Comment by aeiras** [comment]
Happy Friday, Martin! 

Though this ticket is linked to content on the other work being completed on the news page, it has nothing to do with the desired feature. 

Thank you for enabling this follow-up clarification @autumnfound

- The PR belongs to adjusting the template under [EE releases](https://jakarta.ee/release/9.1/) testing it with 9.1 but usable for maybe 9 & even 8 and future releases. 
- 
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/10731224/121739989-46c0e180-cab1-11eb-8f0f-f24efab324a9.png">
The first part of the page shown above tells the viewer:
Version 9.1 NEW RELEASE --- I recommend we insert the in that line after RELEASE **Annoucement**

When we add Announcement, it will link to the formal WG post published under Announcement written for the release. 

Doing so, will keep all the details on each announcement in 1 page. 
The Annoucement section we created will continue to be populated and stuff will no longer be visible or easy to find. Yet when we go to the release, we can find in 1 page all the data that is important about that release. 

I believe that this feature will nicely close the work on the additions we made to the release template.

Now, Tribers don't have the bandwidth to work on this feature. It is our hope that maybe you and the team can push it forward. 

Let me know if this follow up is helpful, Martin!

**Comment by autumnfound** [comment]
> Happy Friday, Martin!
> 
> Though this ticket is linked to content on the other work being completed on the news page, it has nothing to do with the desired feature.
> 
> Thank you for enabling this follow-up clarification @autumnfound
> 
> * The PR belongs to adjusting the template under [EE releases](https://jakarta.ee/release/9.1/) testing it with 9.1 but usable for maybe 9 & even 8 and future releases.
> 
> The first part of the page shown above tells the viewer: Version 9.1 NEW RELEASE --- I recommend we insert the in that line after RELEASE **Annoucement**

If we were to add the link, I imagine it would go after Specifications in the list of buttons below the headline text. This keeps all of our calls to action in a central place and keeps it in an easy-to-find spot!

> When we add Announcement, it will link to the formal WG post published under Announcement written for the release.
> 
> Doing so, will keep all the details on each announcement in 1 page.
> The Annoucement section we created will continue to be populated and stuff will no longer be visible or easy to find. Yet when we go to the release, we can find in 1 page all the data that is important about that release.
> 
> I believe that this feature will nicely close the work on the additions we made to the release template.

This link would be a manual entry rather than an automatic one likely, but still, one that is easy to manage all things considered. Everything in Hugo is templated and thus easy enough to manage!

> 
> Now, Tribers don't have the bandwidth to work on this feature. It is our hope that maybe you and the team can push it forward.
> 
> Let me know if this follow up is helpful, Martin!

If we take on adding functionality, we would just need to check in with the marketing folks to make sure this is where we would want such a feature, or if there is already something in the works that would conflict with this and whatnot. The webdev team is pretty swamped right now with a few bigger projects so this may be prioritized down the line if approved, just as a heads up! 

@karenmcnaughton @TanjaObradovic thoughts?

**Comment by aeiras** [comment]
Martin, you got it.  It is not automatic as every release will have a new blog formal announcement. 
At first i thought about creating a new card for it, as we have for the other items but that is fancy. If i cannot own the work, why would I throw complicated stuff to others. :) simplicity works, start with something and improve. 

You are amazing!

Thank you for the care to explain. +1 to getting extra eyes on the feature.

**Comment by ericpoirier** [comment]
@autumnfound where are we with this request?

**Comment by autumnfound** [comment]
I hadn't committed to working on this as there were other priorities in our pipe. My comment was simply a suggestion on how someone could implement it. I was waiting on feedback/input from site stakeholders/management folks to make sure it doesn't get in the way of other features, and so we could prioritize it for our queue. Once that's done, I could look into doing this if need be! :)
